.github/workflows/release-on-tag.yml

```diff
@@ -0,0 +1,27 @@
+name: Generate Release Notes
+
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - run: npm install github-release-notes -g
+    - name: Write PreRelease Notes
+      env:
+        GREN_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: gren release --override -P --tags=${{ github.event.ref }}
+    - name: Generate Changelog
+      env:
+        GREN_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: gren changelog --override
+    - name: Commit changelog to master
+      run: |
+        echo "How to commit here"
```

Comment:
@DonJayamanne  any idea on how we commit this properly from github actions?

Reply:
https://stackoverflow.com/questions/58257140/unable-to-commit-and-push-back-changes-made-by-github-action-invalid-user?rq=1

Here's sample
```
name: Push commit
on: push
jobs:
  report:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Create report file
        run: date +%s > report.txt
      - name: Commit report
        run: |
          git config --global user.name 'Your Name'
          git config --global user.email 'your-username@users.noreply.github.com'
          git commit -am "Automated report"
          git push

```

Reply:
I tried similar already.. but I will double check

Reply:
If it doesn't work, then lets leave this for now. 
Or we could try doing this via a PR or similar.